language/move-lang/src/typing/core.rs

```diff
@@ -172,6 +172,20 @@ impl Context {
         self.locals.add(var, status).unwrap();
     }
 
+    pub fn is_current_module(&self, m: &ModuleIdent) -> bool {
+        match &self.current_module {
+            Some(curm) => curm == m,
+            None => false,
+        }
+    }
+
+    pub fn is_current_function(&self, m: &ModuleIdent, f: &FunctionName) -> bool {
+        match (&self.current_module, &self.current_function) {
+            (Some(curm), Some(curf)) => curm == m && curf == f,
```

Comment:
Super mega nit: shouldn't this call `is_current_module` to determine whether the current module is `m`?